content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -116,6 +116,11 @@ spec:
           storage: 1Gi
 ```
 
+{{< note >}}
+For this example we use the `ReadWriteOnce` access mode. For production use we
+recommend `ReadWriteOncePod`.
```

Comment:
We [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we) in docs.
```suggestion
This example use the `ReadWriteOnce` access mode, for simplicity. For production use, the Kubernetes
project recommends using the `ReadWriteOncePod` access mode instead.
```